project-base/storefront/public/locales/cs/common.json

```diff
@@ -173,6 +173,7 @@
     "Max allowed limit reached.": "Maximální povolené množství dosaženo.",
     "Menu": "Menu",
     "Message": "Zpráva",
+    "metatagSiteName": "Shopsys Platform",
```

Comment:
Not sure if I completely agree with this approach. I would ask the PO if this is the approach we want to take, rather than loading it from some more dynamic source. I will leave it for you to decide.

Reply:
As the site name should be generic (website/brand/compady), I do not see any source in the administration that could be used other than SEO headline, but that one is specifically for the home page headline/title. IMO we should keep the metatagSiteName in the translation file - it should not change often.